static-assets/components/cstudio-common/common-api.js

```diff
@@ -82,8 +82,6 @@ var nodeOpen = false,
       processing: false,
       compConfProcessing: false,
 
-      UIBuildId: window.UIBuildId,
```

Comment:
Check for other usages of uibuildid nd if possible remove the global for good

Reply:
There are no more usages, and the global was removed in `head.ftl`